tools/jenkins/include/patches/Qt/qt4-sha2.diff

```diff
@@ -214,7 +214,7 @@ diff -burN src.orig/3rdparty/sha2/sha2.cpp src/3rdparty/sha2/sha2.cpp
 + * Please make sure that your system defines BYTE_ORDER.  If your
 + * architecture is little-endian, make sure it also defines
 + * LITTLE_ENDIAN and that the two (BYTE_ORDER and LITTLE_ENDIAN) are
-+ * equivilent.
```

Comment:
please do not touch the patches, because some of them come from external sources

Reply:
sorry, didn't notice it was a `.diff` will remove.